app/components/views/LNPage/NetworkTab/NetworkTab.jsx

```diff
@@ -56,17 +59,116 @@ const QueryNode = ({ getNodeInfo }) => {
   );
 };
 
-const NetworkTab = () => {
-  const { network, nodeInfo, getNodeInfo, getNodeInfoAttempt, tsDate } = useNetworkTab();
+const QueryRoutes = ({
+  nodeID,
+  amount,
+  setNodeID,
+  setAmount,
+  getRoutes
+}) => {
+
+  const nodeChanged = e => {
+    const newNode = e.target.value;
+    if (newNode.length > 66) {
+      return;
+    }
+    setNodeID(newNode);
+  };
 
+  const amtChanged = ({ atomValue }) => {
+    setAmount(atomValue);
+  };
+
+  const requestRoutes = () => {
+    getRoutes(nodeID, amount);
+  };
+
+  return (
+    <div className={styles.queryRoutes}>
+      <div className="memo">
+        <T id="ln.networkTab.queryRoutes.nodeID" m="Node ID" />
+        <TextInput value={nodeID} onChange={nodeChanged} />
+      </div>
+      <div className="value">
+        <T id="ln.networkTab.queryRoutes.value" m="Value" />
+        <DcrInput amount={amount} onChangeAmount={amtChanged} />
+      </div>
+      <KeyBlueButton
+        className={styles.queryRoutesButton}
+        onClick={requestRoutes}>
+      </KeyBlueButton>
+    </div>
+  );
+};
+
+const Tabs = ({ active, set }) => {
+  const QUERY_NODE = 0;
+  const QUERY_ROUTE = 1;
+  return (
+    <div className={styles.tabsWrapper}>
+      <div className={styles.tabs}>
+        <div
+          className={classNames(
+            styles.tabLeft,
+            active === QUERY_NODE
+              ? styles.tabBackgroundSelected
+              : styles.tabBackgroundUnselected
+          )}
+          onClick={() => set(QUERY_NODE)}
+        >
+          Query Node
+        </div>
+        <div
+          className={classNames(
+            styles.tabRight,
+            active === QUERY_ROUTE
+              ? styles.tabBackgroundSelected
+              : styles.tabBackgroundUnselected
+          )}
+          onClick={() => set(QUERY_ROUTE)}
+        >
+          Query Route
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const NetworkTab = () => {
+  const {
+    network,
+    nodeInfo,
+    routesInfo,
+    tsDate,
+    getNodeInfo,
+    getNodeInfoAttempt,
+    getRoutesInfoAttempt,
+    getRoutesInfo
+  } = useNetworkTab();
+  const [activeTab, setActiveTab] = useState(0);
+  const [nodeID, setNodeID] = useState("");
+  const [amount, setAmount] = useState(0);
```

Comment:
Probably best to use `useState(1)` because it seems querying the route with a zero amount always fails.

There's a bug on `DcrInput` that prevents correctly displaying the initial value of 1 atom (https://github.com/decred/decrediton/issues/2682) but once that's fixed this should display correctly.